BankingKataSpecs/AccountSpecs/Features/ToStringOverride.feature

```diff
@@ -0,0 +1,9 @@
+﻿Feature: ToStringOverride
+	In order to get useful information about an account
+	As a math idiot
+	I want to be able to get an informative ToStirng for an account.
+
+@mytag
```

Comment:
They are converted to categories in NUnit so can be used for running parts of your test suite.  @ignore tag also ignores the feature or scenario.

You can also use them for SpecFlow Hooks, for example you can have a BeforeScenario tag and hook that mean for any scenario with that tag the hook is run (this is a brief summary of research online, I've never used hooks or tags).

People also use them for tagging bug reports numbers etc...  You can have multiple tags.